packages/zowe-explorer/src/Profiles.ts

```diff
@@ -1079,11 +1079,25 @@ export class Profiles extends ProfilesCache {
                 cmdArgs[prop] = serviceProfile.profile[prop] ? serviceProfile.profile[prop] : baseProfile.profile[prop];
             }
             if (baseProfile) {
-                cmdArgs.tokenType = serviceProfile.profile.tokenType ? serviceProfile.profile.tokenType: baseProfile.profile.tokenType;
-                cmdArgs.tokenValue = serviceProfile.profile.tokenValue ? serviceProfile.profile.tokenValue: baseProfile.profile.tokenValue;
+                cmdArgs.tokenType = serviceProfile.profile.tokenType
+                    ? serviceProfile.profile.tokenType
+                    : baseProfile.profile.tokenType;
+                cmdArgs.tokenValue = serviceProfile.profile.tokenValue
+                    ? serviceProfile.profile.tokenValue
+                    : baseProfile.profile.tokenValue;
             }
             if (commonApi.getSessionFromCommandArgument) {
-                session = await commonApi.getSessionFromCommandArgument(cmdArgs);
+                if (cmdArgs.tokenType === undefined || cmdArgs.tokenValue === undefined) {
+                    vscode.window.showInformationMessage(
+                        "Base Profile " +
+                            baseProfile.name +
```

Comment:
since the check for tokenType and tokenValue has been updated for alternate profile types as well, this information message shouldn't be based on only base profiles but more general

Reply:
@JillieBeanSim I'm wondering what would be the best way to make this more generic? "Your profile does not have a token. Please login and try again" 😄 

@zdmullen Do you have a suggestion for this?

Reply:
Maybe I can change it to "Profile is not authorized. Please check the connection and try again"

Reply:
I'm not really sure which profile name should I display in the error message. If I make it generic then it is confusing if I have 5 profiles in my tree. I need help here.

Reply:
Hi. Your most recent question refers to the situation in which someone has multiple profiles that use a token, correct? But they only get the error message when they click on the profile, or is it at the activation of the extension? 

If I had to suggest an error message now, I would go with "Your profile no longer has a valid token. Please log in and try again."

Reply:
This appears during initialization. And if I have multiple profiles in the tree, I'm not sure which one has no valid token. Unless I  just remove the error message here and let the initialization continue then there will still be an error message once the profile is used.

Reply:
I like that, with the error when the profile is used vs during initialization

Reply:
ok so I'll just remove this error message and add it to the logs for record keeping.